prefect_kubernetes/worker.py

```diff
@@ -517,6 +519,14 @@ class KubernetesWorkerVariables(BaseVariables):
         default=None,
         description="The Kubernetes cluster config to use for job creation.",
     )
+    tcp_keepalive: bool = Field(
+        default=True,
+        description=(
+            "Maintain connections to the Kubernetes API by sending "
+            "keep-alive messages. Recommended when using cloud load "
+            "balancers or firewalls."
+        ),
+    )
```

Comment:
Is there any downside to always having this enabled? If we want to give users the option to turn this off, it should be controlled via an environment variable rather than a field in the base job template since this setting controls worker behavior and doesn't change the behavior of jobs created for flow runs.

Reply:
Good point! It was configurable in prefect 1, but I couldn't figure out if it was on or off by default. I think having it on by default is a good idea given how prevalent this issue is on Azure, and how difficult it might be to track down which thing you'd need to turn on for the worker to stop failing. If we convert it to an env var, what would be a good way to have it be on by default? Exposing it through the helm chart?

Reply:
We might be able to do something similar to what we're doing with `PREFECT_KUBERNETES_WORKER_STORE_PREFECT_API_IN_SECRET` i.e.:

```python        
if os.environ.get(
        "PREFECT_KUBERNETES_WORKER_ADD_TCP_KEEPALIVE", "true"
).strip().lower() in ("true", "1"):
  # add keepalive
```